src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -142,6 +144,18 @@ public VerboseProcess(final Process prc, final Level stdout,
         if (stderr == null) {
             throw new IllegalArgumentException("stderr LEVEL can't be NULL");
         }
+        final String format = "%s LEVEL can't be set to ALL because it is "
```

Comment:
@prondzyn this looks weird. Just use the text where it's required. No need to do this complex formatting, etc.